### PR TITLE
Add support for Docker in Camyks

### PR DESCRIPTION
**Pull request by Oldkarkass**
- Add docker file to support Docker for CaMyKs
- Container runs on Debian 8 / Apache 2.4/ PHP 5.6 / MySQL 5.5

### CODE REVIEW

**Comment by Oldkarkass** [comment]
- Need rebase on dev
- Need to add apache rewrite

File: README.md [hunk 1]
```diff
@@ -32,6 +32,23 @@ Edit `site.php.inc` configuration file located in the etc folder.
 
 Open `index.php` or `admin.php` from your web browser to finalise your website installation.
 
+## Building the docker container
```
Comment (Dj1b) [inline README.md]: Add docker part in one single chapter, and 2 sub-chapters

File: docker/site.php.inc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ */
+
+/* Site configuration object */
+$conf = array();
+/* CMS name : CaMykS */
+$conf['cms'] = 'CaMykS';
+/* CMS version */
+$conf['cms_version'] = '1.0b';
+/* path to CaMykS folder */
+$conf['engine_path'] = '/app/camyks';
```
Comment (Dj1b) [inline docker/site.php.inc]: Simply indicate "camyks" to have a single site installation

File: docker/site.php.inc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+/* database config */
+$conf['database_driver'] = 'mysql';
+$conf['database_name'] = 'camyks';
+$conf['database_host'] = 'localhost';
+$conf['database_login'] = 'camyks';
+$conf['database_pwd'] = 'camyks';
+/* site admin ssl : default | auto | forced | none */
+$conf['admin_ssl'] = 'default';
+/* site admin default language : fr, en */
+$conf['admin_default_language'] = 'fr';
```
Comment (Dj1b) [inline docker/site.php.inc]: Maybe set languages to "en"

**Comment by Oldkarkass** [comment]
rebased the PR with today dev branch

**Pull request merged**